export const throwError = (): never => {
  throw new Error('Mensagem de erro');
}
